Formula/goto.rb

```diff
@@ -0,0 +1,17 @@
+class Goto < Formula
+  desc "Shell tool for navigation to aliased directories with auto-completion"
+  homepage "https://github.com/iridakos/goto"
+  url "https://github.com/iridakos/goto/archive/v1.2.2.tar.gz"
+  sha256 "611ef7b82f6798ef83db0473811740c7f1ff5a15bb4ff52a1e5841bac87ba7f5"
+
+  bottle :unneeded
+
+  def install
+    bash_completion.install "goto.sh" => "goto"
```

Comment:
If the name of the script in the package is `goto.sh` then we should keep that, not rename it. Principle of least surprise.

Reply:
Correct, brewing fix.